cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -923,6 +927,64 @@ public KafkaBrokerConfigurationBuilder withTieredStorage(String clusterName, Tie
         return this;
     }
 
+    /**
+     * Configures the quotas based on the type of the plugin - {@link QuotasPluginKafka}, {@link QuotasPluginStrimzi}
+     *
+     * @param clusterName    Name of the cluster
+     * @param quotasPlugin   Configuration of the quotas plugin
+     * @return  Returns the builder instance
+     */
+    public KafkaBrokerConfigurationBuilder withQuotas(String clusterName, QuotasPlugin quotasPlugin) {
+        if (quotasPlugin != null) {
+            // for the built-in Kafka quotas plugin we don't need to configure anything
+            if (quotasPlugin instanceof QuotasPluginStrimzi quotasPluginStrimzi) {
+                printSectionHeader("Quotas configuration");
+                configureQuotasPluginStrimzi(clusterName, quotasPluginStrimzi);
+                writer.println();
+            }
+        }
+
+        return this;
+    }
+
+    /**
+     * Configures Strimzi quotas plugin
+     *
+     * @param clusterName           Name of the cluster
+     * @param quotasPluginStrimzi   Strimzi quotas plugin configuration
+     */
+    private void configureQuotasPluginStrimzi(String clusterName, QuotasPluginStrimzi quotasPluginStrimzi) {
+        // add Kafka broker's and CruiseControl's user to the excluded principals
+        List<String> excludedPrincipals = new ArrayList<>(List.of(
+            String.format("User:CN=%s,O=io.strimzi", KafkaResources.kafkaComponentName(clusterName)),
+            String.format("User:CN=%s,O=io.strimzi", CruiseControlResources.componentName(clusterName))
+        ));
+
+        writer.println(String.format("%s=%s", "client.quota.callback.class", "io.strimzi.kafka.quotas.StaticQuotaCallback"));
+
+        // configuration of Admin client that will check the cluster
+        writer.println(String.format("client.quota.callback.static.kafka.admin.bootstrap.servers=%s", clusterName + "-kafka-brokers:9091"));
+        writer.println("client.quota.callback.static.kafka.admin.security.protocol=SSL");
+        writer.println("client.quota.callback.static.kafka.admin.ssl.keystore.location=/tmp/kafka/cluster.keystore.p12");
+        writer.println("client.quota.callback.static.kafka.admin.ssl.keystore.password=${CERTS_STORE_PASSWORD}");
+        writer.println("client.quota.callback.static.kafka.admin.ssl.keystore.type=PKCS12");
+        writer.println("client.quota.callback.static.kafka.admin.ssl.truststore.location=/tmp/kafka/cluster.truststore.p12");
+        writer.println("client.quota.callback.static.kafka.admin.ssl.truststore.password=${CERTS_STORE_PASSWORD}");
+        writer.println("client.quota.callback.static.kafka.admin.ssl.truststore.type=PKCS12");
```

Comment:
Sorry, just to understand it correctly: 

Should I rewrite it to use the variables there and `String.format()` so something like:
`String.format("client.quota.callback.static.kafka.admin.ssl.keystore.password=%s", PLACEHOLDER_CERT_STORE_PASSWORD);` for all the places where such variable (or method for getting such value) exists?

Or you meant to have also the config options in some class (I had such before, but it was changed after some comment)?

Because CC config and other configs have it different, so I'm just thinking what is the correct way how to proceed with this.

Reply:
Yes, you should use the placeholder, you should use the same way to configure the bootstrap server etc.

Reply:
Ok, now - once I push the changes - it will be (hopefully) fine, thanks 😄